app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{"
+            f"Name: {self.name}, "
+            f"Health: {self.health}, Hidden: {self.hidden}"
+            f"}}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden is False:
+            self.hidden = True
+        else:
+            self.hidden = False
```

Comment:
Write this method in one line as mentioned [in the checklist](https://github.com/oleksashcherbakov/py-herbivores-and-carnivores/blob/develop/checklist.md#code-efficiency).